website/server/docgenHelpers.js

```diff
@@ -1,5 +1,12 @@
 "use strict";
-var b = require('react-docgen/node_modules/recast').types.builders;
+var b;
+try {
+  // npm 3 probably
+  b = require('recast').types.builders;
```

Comment:
That would probably be the better option :) I didn't actually look at any of the code here to figure out *why* it is like this, just made it run. #lazy

But now that I'm actually looking at the rest of this, we can just remove this whole require. `recast` is getting passed into the APIs below (you can see `b` get pulled off there so this one isn't even getting used).